src/i18n/en.json

```diff
@@ -58,24 +58,28 @@
   },
   "reports_box": {
     "to": "to",
-    "visits_description": "This report shows the total number of visits in the selected time period",
-    "visits_with_email": "Visits with emails",
-    "visits_without_emails": "Visits without emails"
+    "visits_description": "Total number of users who visited your Website and whose Email has not been identified by Doppler. If a user entered several times, only one will be counted.",
```

Comment:
`visits_description` is not used anymore, right? in that case, could you remove it?